Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Spawning/SpawnNodeable.h

```diff
@@ -39,5 +41,8 @@ namespace ScriptCanvas::Nodeables::Spawning
 
     private:
         AzFramework::Scripts::SpawnableScriptMediator m_spawnableScriptMediator;
+
+        AZStd::vector<AzFramework::Scripts::SpawnableScriptAssetRef> m_dummy;
```

Comment:
Please add a comment explaining what this is and why it's necessary. Possibly consider a more descriptive name than "m_dummy" too. Is it a dummy because it's a placeholder? Unused? A container of references just used to avoid link optimizations? I feel like there's probably a name that describes the need for this better.

Reply:
oh, this isn't needed anymore, it was an experiment on my part. Thanks for calling it out! I'll remove it